src/components/Characters/ConfusionMarks.js

```diff
@@ -1,22 +1,33 @@
 import React from 'react';
 import styled from 'styled-components';
+import { useSpring, animated } from 'react-spring';
 
 import { zIndexArtworkSecondaryDecor } from '../../styles/designTokens';
 
-function ConfusionMarksArtwork({ className }) {
-  return (
-    <svg xmlns="http://www.w3.org/2000/svg" width="29" height="27" viewBox="0 0 29 27" className={className}>
-      <path
-        fill="#222"
-        d="M19.115 25.84c.913 0 1.711-.608 2.167-1.065.304-.304.38-.798.38-1.445 0-.761-2.358-2.358-3.308-2.358-1.255 0-1.559.837-1.863 1.787-.076.228-.152.875-.152 1.065 0 .418.532.799.798 1.217.077.115.99.799 1.978.799zM.524 6.327c-.247-.871-.72-2.933.507-3.447.531-.222.807-.026 1.1-.071.667-.084.764.148 1.151.143.968-.015 1.328.38 1.536.878.402.962.084 2.342.184 3.235.451 4.059.996 7.689.957 11.603-.002.274-.129.716-.56.897-.199.083-.426.1-.7.097-.401-.027-.664-.19-.766-.341-.466-.741-1.242-.455-1.52-1.118-.055-.132-.072-.359-.103-.619-.07-.633-.181-.899-.284-1.518-.542-3.437-.732-7.06-1.502-9.739zm16.614 9.284c-.152-.266-.418-.875-.418-1.331a.92.92 0 0 1 .266-.685c.114-.114.076-.19.228-.456 1.445-2.51 3.612-4.563 4.753-5.78 1.977-2.13 2.738-3.08 2.738-3.46 0-.418-.418-.951-.989-.951-.19 0-.418.076-.608.19-.761.456-.913.761-.913 1.293 0 .266.038.532.038.913 0 .38-.076.951-1.217.951-1.065 0-1.711-.532-2.13-1.065-.228-.304-.266-1.065-.266-1.483 0-1.065.38-2.015.875-2.472C20.37.476 21.51.058 22.879.058c1.483 0 2.51.266 3.613.913 1.521.913 2.091 2.129 2.091 3.08 0 1.635-.532 2.472-4.145 5.97-2.206 2.13-3.613 4.145-3.613 4.373 0 .19.342.228.799.228.57 0 1.103-.076 1.255-.076.913 0 1.673.228 2.358.646.19.114.609.228.837.799.076.19.228.532.228.799 0 .342-.19.608-.875.722-.266.038-.685.266-1.217.266-2.129 0-4.031-.19-5.59-.837-.721-.303-1.026-.531-1.482-1.33zM5.66 22.612c.724-.225 1.184.129 1.378.593.083.199.192.738.234 1.305.045.76-.056 1.543-.753 1.834-.365.153-.716.339-1.014.463-.431.18-.857.281-1.341.055a2.857 2.857 0 0 1-1.385-1.447c-.18-.431-.239-.758-.187-1.286.027-.401-.308-.923.587-1.298 1.062-.444 1.544.056 2.481-.219z"
-      />
-    </svg>
-  );
-}
-
-const ConfusionMarks = styled(ConfusionMarksArtwork)`
+const Wrapper = styled(animated.span)`
   position: absolute;
   z-index: ${zIndexArtworkSecondaryDecor};
+  transform-origin: 50% 200%;
 `;
 
+function ConfusionMarks({ className }) {
```

Comment:
Just spread props instead of single className:
```
function ConfusionMarks(props) {
  ...
  return <Wrapper style={animationStyles} {...props} />...</Wrapper>
}
```